vassal-app/src/main/java/VASSAL/i18n/Resources.java

```diff
@@ -327,28 +369,21 @@ public static String getString(ResourceBundle bundle, String id) {
     return s;
   }
 
-  public static String getString(String id, Object... params) {
-    return getInstance().getBundleForKey(id).getString(id, params);
-  }
-
-  protected static final String BASE_BUNDLE = "VASSAL.properties";
-  protected static final String EN_BUNDLE = "VASSAL_en.properties";
-
   /**
    * Custom Class Loader for loading VASSAL property files.
    * Check first for files in the VASSAL home directory.
    *
    * @author Brent Easton
    */
-  public class VassalPropertyClassLoader extends ClassLoader {
+  public static class VassalPropertyClassLoader extends ClassLoader {
```

Comment:
This would cause an exception if anyone subclassed it but didn't recompile. (Odds are that no one will have, though.)

Reply:
True, best to leave it. Reverted.